### ISSUE TITLE
Data release: v20

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-analysis#1048]
#### What data file(s) does this issue pertain to?
```
pbta-histologies.tsv
pbta-snv-scavenged-hotspots.maf.tsv.gz
pbta-cnv-consensus.seg.gz
consensus_seg_annotated_cn_autosomes.tsv.gz
consensus_seg_annotated_cn_x_and_y.tsv.gz
```

#### What release are you using?

v19

#### Put a link to the relevant section of the [OpenPBTA-manuscript](https://github.com/AlexsLemonade/OpenPBTA-manuscript/) here.
NA

#### Put your question or report your issue here.
Once #961 and #987 are merged and subtyping tickets are finished, we will do a final release (v20) prior to publication submission. The above files will be updated/added.

**Comment by jharenza** [comment]
v20 ADAPT histologies file pull is located here: `s3://d3b-bix-dev-data-bucket/hgg-dmg-integration/20210719-data/` @kgaonkar6

**Comment by jaclyn-taroni** [comment]
Closed via #1118
